index.js

```diff
@@ -138,6 +138,18 @@ Hook.prototype.initialize = function initialize() {
     catch (e) { return this.log(this.format(Hook.log.binary, binary), 0); }
   }, this);
 
+  // in GUI clients node and npm are not in the PATH
+  // so get node binery PATH, add it to the PATH list
+  // and try again
+  if (!this.npm) {
```

Comment:
This also highlights another mistake in the code. As it shouldn't even get there as there might not be a git or npm binary.